project.js

```diff
@@ -464,13 +467,22 @@ function processFileList() {
 // Empty and hide the failed project list
 // Reset the download button
 function hideEmptyProjectList() {
+    // Clear the list of excluded projects
     $('#removed-projects')
         .html(' ')
         .addClass('hidden');
 
-    $('#downloadProjBtn')
+    // Update the button to pre-download defaults
+    $('#downloadProjectButton')
         .removeClass('btn-success')
         .addClass('btn-primary')
         .html('Download My Projects')
         .on('click', startProjectsDownload);
-}
\ No newline at end of file
+
+    // Reset the exporter variables
+    theFileList = [];
```

Comment:
Resetting all of these globals will leave the page in a known state after the first download is completed.